src/theme/createSaleorTheme/createSaleorTheme.ts

```diff
@@ -58,6 +58,11 @@ export const createTheme = (colors: SaleorThemeColors): SaleorTheme =>
           fontSize: "1.3rem",
         },
       },
+      MuiDialog: {
+        paper: {
+          overflowY: undefined,
```

Comment:
What does `undefined` accomplish here? What's the CSS prop value outcome? If it's "initial" then it would be best to type it manually.

Reply:
It totally removes value from stylesheet, in my opinion it reduces css bloat